### PR TITLE
Fix examples

### PR DESCRIPTION
**Pull request by capsert**

### CODE REVIEW

File: examples/1-new-payment.py [hunk 1]
```diff
@@ -58,7 +59,7 @@ def main():
         #
         return flask.redirect(payment.checkout_url)
 
-    except mollie.api.Error as e:
+    except Error as e:
```
Comment (whyscream) [inline examples/1-new-payment.py]: No single-letter variables: replace `e` with `err`

File: examples/10-customer-payment-history.py [hunk 2]
```diff
@@ -50,11 +51,13 @@ def main():
         # See: https://www.mollie.com/nl/docs/reference/customers/list-payments
         #
         params = {
-            'from': 0,
-            'count': amount_of_payments_to_retrieve,
+            'limit': amount_of_payments_to_retrieve,
         }
         payments = mollie_client.customer_payments.with_parent_id(customer_id).all(**params)
 
+        import ipdb
```
Comment (whyscream) [inline examples/10-customer-payment-history.py]: debugging statement

File: examples/10-customer-payment-history.py [hunk 3]
```diff
@@ -65,7 +68,7 @@ def main():
 
         return body
 
-    except mollie.api.error as e:
+    except Error as e:
```
Comment (whyscream) [inline examples/10-customer-payment-history.py]: No single-letter variables: replace `e` with `err`

File: examples/11-refund-payment.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    api_key = os.environ.get('MOLLIE_API_KEY', 'test_test')
+    mollie_client = Client()
+    mollie_client.set_api_key(api_key)
+
+    body = ''
+    payment_id = ''
+
+    payments = mollie_client.payments.all()
+
+    body += '<p>No payment ID specified. Attempting to retrieve all payments and grabbing the first.</p>'
```
Comment (whyscream) [inline examples/11-refund-payment.py]: Dit suggereert dat je wel een payment id kan specificeren, bijvoorbeeld via een querystring parameter. Dat is echter helemaal niet mogelijk.

File: examples/2-webhook-verification.py [hunk 5]
```diff
@@ -58,7 +59,7 @@ def main():
             #
             return 'Cancelled'
 
-    except mollie.api.error as e:
+    except Error as e:
```
Comment (whyscream) [inline examples/2-webhook-verification.py]: No single-letter variables: replace `e` with `err`

File: examples/4-ideal-payment.py [hunk 6]
```diff
@@ -44,44 +44,45 @@ def main():
 
             if flask.request.form['issuer']:
                 issuer_id = str(flask.request.form['issuer'])
+        import ipdb
```
Comment (whyscream) [inline examples/4-ideal-payment.py]: debug statement?

File: examples/4-ideal-payment.py [hunk 7]
```diff
@@ -44,44 +44,45 @@ def main():
...
-            #
-            # Send the customer off to complete the payment.
-            #
-            return flask.redirect(payment.checkout_url)
+        #
+        # Send the customer off to complete the payment.
+        #
+        return flask.redirect(payment.checkout_url)
 
     except Error as e:
```
Comment (whyscream) [inline examples/4-ideal-payment.py]: No single-letter variables: replace `e` with `err`

File: examples/5-payments-history.py [hunk 8]
```diff
@@ -32,7 +33,7 @@ def main():
 
         return body
 
-    except mollie.api.error as e:
+    except Error as e:
```
Comment (whyscream) [inline examples/5-payments-history.py]: No single-letter variables: replace `e` with `err`

File: examples/6-list-activated-methods.py [hunk 9]
```diff
@@ -40,7 +40,7 @@ def main():
 
         return body
 
-    except mollie.api.error as e:
+    except Error as e:
```
Comment (whyscream) [inline examples/6-list-activated-methods.py]: No single-letter variables: replace `e` with `err`

File: examples/7-new-customer.py [hunk 10]
```diff
@@ -31,7 +32,7 @@ def main():
 
         return "Created new customer '%s' (%s)" % (customer.name, customer.email)
 
-    except mollie.api.error as e:
+    except Error as e:
```
Comment (whyscream) [inline examples/7-new-customer.py]: No single-letter variables: replace `e` with `err`

File: examples/1-new-payment.py [hunk 11]
```diff
@@ -58,7 +59,7 @@ def main():
...
         return 'API call failed: ' + str(e)
```
Comment (whyscream) [inline examples/1-new-payment.py]: Use `%s` and don't cast exception to string.

File: examples/8-list-customers.py [hunk 12]
```diff
@@ -17,20 +18,21 @@ def main():
...
-            'count': amount_of_customers_to_retrieve,
+            'limit': amount_of_customers_to_retrieve,
         }
 
         #
         # Get the latest 20 customers
         #
         # See: https://www.mollie.com/nl/docs/reference/customers/list
         #
+        import ipdb
```
Comment (whyscream) [inline examples/8-list-customers.py]: debug statement?

File: examples/app.py [hunk 13]
```diff
@@ -13,7 +13,8 @@
     '7-new-customer',
     '8-list-customers',
     '9-create-customer-payment',
-    '10-customer-payment-history'
+    '10-customer-payment-history',
+    '11-refund-payment'
```
Comment (whyscream) [inline examples/app.py]: add trailing comma

File: mollie/api/objects/list.py [hunk 14]
```diff
@@ -29,7 +29,7 @@ def __next__(self):
         try:
             item = self['_embedded'][self.get_object_name()][self.current]
             return self.object_type(item)
-        except IndexError:
+        except (IndexError, KeyError):
```
Comment (whyscream) [inline mollie/api/objects/list.py]: when does a KeyError happen here?

File: mollie/api/objects/payment.py [hunk 15]
```diff
@@ -152,16 +152,16 @@ def can_be_refunded(self):
     @property
     def get_amount_refunded(self):
```
Comment (whyscream) [inline mollie/api/objects/payment.py]: property should be named simply 'amount_refunded'

File: mollie/api/objects/payment.py [hunk 16]
```diff
@@ -152,16 +152,16 @@ def can_be_refunded(self):
...
         try:
-            return float(self._get_property('amountRefunded'))
-        except TypeError:
-            return 0.0
+            return self['amountRefunded']
+        except KeyError:
+            return '0.0'
 
     @property
     def get_amount_remaining(self):
```
Comment (whyscream) [inline mollie/api/objects/payment.py]: prperty should be named simply `amount_remaining`

File: mollie/api/objects/payment.py [hunk 17]
```diff
@@ -152,16 +152,16 @@ def can_be_refunded(self):
...
         try:
-            return float(self._get_property('amountRemaining'))
-        except TypeError:
-            return 0.0
+            return self['amountRemaining']
```
Comment (whyscream) [inline mollie/api/objects/payment.py]: Use `self._get_property(...) or 0.0`. Also, why is this a string suddenly?

Comment (capsert) [reply mollie/api/objects/payment.py]: https://docs.mollie.com/reference/v2/payments-api/get-payment
Here is documented that `amountRefunded` and `amountRemaining` are returned as a string

**Pull request merged**